pydantic/functional_serializers.py

```diff
@@ -32,11 +51,32 @@ def __get_pydantic_core_schema__(self, source_type: Any, handler: GetCoreSchemaH
 
 @slots_dataclass(frozen=True)
 class WrapSerializer:
+    """
+    This class is used to wrap a serializer function and bind it with requested parameters for later use in
+    schema building.
```

Comment:
I think this description is not right. I haven't figured out a better one yet, but I'm commenting now to make sure I come back to it. @adriangb might be able to think of one before me...

Reply:
That sounds good David